bricks/basic/src/card-box/host-context.css

```
@@ -0,0 +1,3 @@
+html[data-theme="dark"],
+html[data-theme="dark-v2"] {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)